daemon/openrazer_daemon/hardware/mouse_mat.py

```diff
@@ -24,3 +24,41 @@ class RazerFirefly(__RazerDeviceBrightnessSuspend):
         "side_img": "https://assets.razerzone.com/eeimages/products/21936/fireflycloth-gallery-6.png",
         "perspective_img": "https://assets.razerzone.com/eeimages/products/21936/fireflycloth-gallery-3.png"
     }
+
+
+class RazerGoliathus(__RazerDeviceBrightnessSuspend):
+    """
+    Class for the Razer Goliathus (2018)
+    """
+    USB_VID = 0x1532
+    USB_PID = 0x0C01
+    HAS_MATRIX = True
+    MATRIX_DIMS = [1, 1]
+    METHODS = ['get_device_type_mousemat', 'set_static_effect', 'set_spectrum_effect',
+               'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row']
+
+    RAZER_URLS = {
```

Comment:
Can be amended in line with aeabb468fe1ec700ac4cc1647cd09b93a26daeed like so:

```suggestion
    DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1394/1394_goliathus_chroma.png"

    # Deprecated - RAZER_URLS be removed in future.    
    RAZER_URLS = {
```

Reply:
I just replaced these with DEVICE_IMAGE.